File: exe2.java
public class exe2 {
    private String marca;
    private String modelo;
    private int ano;
    private int velocidade;

    public exe2(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
        this.velocidade = 0; 
    }

    public void acelerar(int incremento) {
        this.velocidade += incremento;
        System.out.println("Acelerando... Velocidade atual: " + this.velocidade + " km/h");
    }

    public void frear(int decremento) {
        this.velocidade -= decremento;
        if (this.velocidade < 0) {
            this.velocidade = 0; 
        }
        System.out.println("Freando... Velocidade atual: " + this.velocidade + " km/h");
    }

    public void exibirVelocidade() {
        System.out.println("Velocidade atual: " + this.velocidade + " km/h");
    }

    public static void main(String[] args) {

        exe2 carro = new exe2("Toyota", "Corolla", 2020);

        carro.acelerar(50); 
        carro.frear(20);     
        carro.exibirVelocidade(); 
    }
}

File: exe2.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano
        self.velocidade = 0

    def acelerar(self, incremento):
        self.velocidade += incremento
        print(f"Acelerando... Velocidade atual: {self.velocidade} km/h")

    def frear(self, decremento):
        self.velocidade -= decremento
        if self.velocidade < 0:
            self.velocidade = 0 
        print(f"Freando... Velocidade atual: {self.velocidade} km/h")

    def exibir_velocidade(self):
        print(f"Velocidade atual: {self.velocidade} km/h")


carro = Carro("Toyota", "Corolla", 2020)

carro.acelerar(50)
carro.frear(20)     
carro.exibir_velocidade()  
